omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
Btw, dependency of `OmisePaymentModuleFrontController` doesn't increase.
It's what "ControllerCore" provided for us (from what we extended `ModuleFrontController`).

Instead, if we call `Currency::xxx()`, `Cart::xxx()` inside another class that doesn't extend `ModuleFrontController` directly, that's increase dependency.